Makefile

```diff
@@ -248,9 +248,10 @@ kind-load: $(KIND) #EXHELP Loads the currently constructed images into the KIND
 
 .PHONY: kind-deploy
 kind-deploy: export MANIFEST := ./operator-controller.yaml
+kind-deploy: export DEFAULT_CATALOG := ./catalogd/config/base/default/clustercatalogs/default-catalogs.yaml
```

Comment:
If I change this file, I want to test it out.
So, we cannot use the release one.
Otherwise, we would only get issues related to changes on this file after we do a release